Themes/default/Xml.template.php

```diff
@@ -107,7 +107,7 @@ function template_modifytopicdone()
 	}
 	else
 		echo '
-		<error in_subject="', $context['message']['error_in_subject'] ? '1' : '0', '"><![CDATA[', cleanXml(implode('<br />', $context['message']['errors'])), ']]></error>';
+		<error in_subject="', $context['message']['error_in_subject'] ? '1' : '0', '"><![CDATA[', cleanXml(implode('<br>', $context['message']['errors'])), ']]></error>';
```

Comment:
This is an implode, so maybe it was also correct?

Reply:
this one I would think is fine as it outputing a break before error list (could be wrong just how I read the code)